lib/Conversion/FIRRTLToRTL/LowerToRTL.cpp

```diff
@@ -1634,8 +1635,10 @@ LogicalResult FIRRTLLowering::visitDecl(MemOp op) {
     case MemOp::PortKind::Read: {
       // Add delays for non-zero read latency
       SmallVector<ReadPipeElement> readPipe;
-      readPipe.push_back({portWires["en"], portWires["addr"]});
-      Value enReg, addrReg;
+      readPipe.push_back({portWires["en"], portWires["addr"],
+                          builder->create<sv::ReadInOutOp>(portWires["en"]),
+                          builder->create<sv::ReadInOutOp>(portWires["addr"])});
```

Comment:
Is the construction order stable/defined for the two `ReadInOutOp`s here? (I don't know and I already was bitten by this for function parameter ordering...)